contrib/kind.sh

```diff
@@ -981,9 +981,11 @@ install_multus() {
 }
 
 install_mpolicy_crd() {
+
   echo "Installing multi-network-policy CRD ..."
-  mpolicy_manifest="https://raw.githubusercontent.com/k8snetworkplumbingwg/multi-networkpolicy/master/scheme.yml"
-  run_kubectl apply -f "$mpolicy_manifest"
+  pushd ${MANIFEST_OUTPUT_DIR}
```

Comment:
why is this change happening? generally speaking we don't want to maintain a copy of upstream CRDs in our repo, we prefer to direct install.

Reply:
oh so now I see what @npinaeva was saying.. I agree with her, we cannot maintain a local copy like that of an unreleased version.

what we can do is to have a https://github.com/k8snetworkplumbingwg/multi-networkpolicy/tags tag request while @girishmg is working on getting the release cut.

Have a v1beta2.alpha or whatever will be a pre-released version is, put this behind a feature gate that is off by default because we don't want to switch this on d/s for us, so its only fair that new features go behind feature gates -> we can discuss this more on u/s meeting or have a separate meeting for this if needed